### DISCUSSION TITLE
Unable to calculate interaction forces

### DISCUSSION THREAD

**Discussion by qinyj666** [deepmodeling/deepmd-kit#3909] [Q&A]
I tried to calculate the interaction force and the interaction energy between two groups with the command ‘compute 111 c group/group nacl’; the final outputs were both zero.
Step TotEng PotEng KinEng Temp Press Density c_Tnacl c_111 c_111[1] c_111[2] c_111[3]
0 -17249.992 -17360.525 110.53294 280 763.98422 0.44456891 277.8397 0 0 0 0
10000 -17136.622 -17250.113 113.4906 287.49229 -493.94182 0.44456891 281.48387 0 0 0 0
20000 -17140.72 -17253.361 112.64031 285.33835 -222.46923 0.44456891 282.63192 0 0 0 0
30000 -17148.698 -17259.099 110.40095 279.66566 -673.49257 0.44456891 280.52655 0 0 0 0

**Comment by wanghan-iapcm** [comment]
The DP models are many-body potential models. they do not support pairwise decomposition thus one is not able to compute the interaction between group of atoms.
